invoicer_project/clients/templates/clients/clients_list.html

```diff
@@ -0,0 +1,126 @@
+            <div class="col-2" id="navigation_hamburger_button_and_logo">
+                <div class="navbar-header fixed-brand">
+                    <button
+                        id="menu-toggle-2"
+                        type="button"
+                        data-bs-toggle="collapse"
+                        data-bs-target="#navbarSupportedContent"
+                        aria-controls="navbarSupportedContent"
+                        aria-expanded="false"
+                        aria-label="Toggle navigation">
+                        <i class="material-icons" style="font-size: 36px; color: #767680">menu</i>
+                    </button>
+                    <a href="index.html">
+                        <img src="{% static 'img/logo.svg' %}" alt="logo" width="146" height="45"/>
+                    </a>
+                </div>
+                <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav">
+                        <li class="active">
+                            <button
+                                class="navbar-toggler"
+                                id="menu-toggle"
+                                type="button"
+                                data-bs-toggle="collapse"
+                                data-bs-target="#navbarSupportedContent"
+                                aria-controls="navbarSupportedContent"
+                                aria-expanded="false"
+                                aria-label="Toggle navigation">
+                                <span class="navbar-toggler-icon"></span>
+                            </button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="d-flex flex-wrap flex-row align-items-center col-8">
+            <div class="search">
+                <button type="submit" class="search_button">
+                    <img src="{% static 'img/search.svg' %}"/>
+                </button>
+                <input id="search_bar" type="text" class="search_term"
+                    placeholder="Search item...">
+
+            </div>
+            <div class="sort_dropdown">
+                <button class="sort_drop_btn"><span class="material-symbols-outlined">sort</span></button>
+                <div class="sort_dropdown-content">
+                  <a>A-Z by name</a>
+                </div>
+              </div>
+            <md-fab-extended icon="add" label="Add client"></md-fab-extended>
+        </div>
+            <div class="col-2" id="user_name_and_icon_column">
+                <p id="user_name">Name Surname</p>
+                <i class="material-icons">account_circle</i>
+            </div>
+        </nav>
+        <div id="wrapper" class="toggled-2">
+            <div id="sidebar-wrapper">
+                <ul class="sidebar-nav nav-pills nav-stacked" id="menu">
+                    <li>
+                        <a href="#"
```

Comment:
let's format this code

Reply:
Sorry, I haven't understood what do you mean. Could you explain in detail?

Reply:
Done.